dae-alright/src/components/Modal/CustomModal/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import Modal from "react-bootstrap/Modal";
+import { Button } from "react-bootstrap";
+import "./index.scss";
+
+const CustomModal = props => {
+  const [show, setShow] = useState(false);
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  return (
+    <>
+      {/* <Button variant="primary" onClick={handleShow}>
```

Comment:
We agreed to remove this. No unwanted code, please.

Reply:
It's there so that others can test, I would remove it as soon as I got a reply from them.